modules/network-observability-flow-filter-parameters.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: REFERENCE
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+[id="network-observability-flowcollector-flowfilter-parameters_{context}"]
+= Flow filter configuration parameters
+The flow filter rules consist of mandatory and optional parameters.
```

Comment:
```suggestion
The flow filter rules consist of required and optional parameters.
```
The change to "required" here better matches the table below it. Just a thought.

Reply:
Great detail 👍